docker-compose.yml

```
@@ -1,63 +1,67 @@
 version: '3'
+
+x-env-generic-backend: &x-env-generic-backend
+  TZ: Europe/Stockholm
+  WHARF_API_URL: http://api:8080
+  ALLOW_CORS: "YES"
+
 services:
   proxy:
     image: amd64/traefik:v2.5
     ports:
     - "5000:8081"
     volumes:
     - ./wharf-docker-compose/traefik:/etc/traefik # Traefik configuration file
-  web:
-    image: quay.io/iver-wharf/wharf-web
+#  web:
+#    image: quay.io/iver-wharf/wharf-web
```

wharf-web is disabled in the docker-compose.build.yml. In this file, wharf-web needs to be enabled so it's easy to spin up a local instance of Wharf without needing any other repos than wharf-docker-compose

```suggestion
  web:
    image: quay.io/iver-wharf/wharf-web
```